examples/configuration.rs

```diff
@@ -61,19 +115,23 @@ valkey_module! {
     configurations: [
         i64: [
             ["i64", &*CONFIGURATION_I64, 10, 0, 1000, ConfigurationFlags::DEFAULT, Some(Box::new(on_configuration_changed))],
+            ["reject-i64", &*CONFIGURATION_REJECT_I64, 10, 0, 1000, ConfigurationFlags::DEFAULT, Some(Box::new(on_configuration_changed)), Some(Box::new(on_i64_config_set::<ValkeyString, ValkeyGILGuard<i64>>))],
```

Comment:
tiny nit:  could you stick to snake_case just to be consistent in all 4 examples